src/plugins/googledrive/admin/src/components/PluginIcon/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ *
+ * PluginIcon
```

Comment:
Similar feedback here, do we need a comment, considering this code is pretty much self documented as it relates to this comment?